delta/kernel/src/main/scala/ch/epfl/bluebrain/nexus/delta/kernel/utils/ClasspathResourceLoader.scala

```diff
@@ -124,7 +121,7 @@ class ClasspathResourceLoader private (classLoader: ClassLoader) {
 }
 
 object ClasspathResourceLoader {
-  private[utils] val handleBars = new Handlebars().`with`(EscapingStrategy.NOOP)
+  private val handlebarsExpander = new HandlebarsExpander
```

Comment:
What was the issue here ?

Reply:
I had previously added something else to the handlebars instance which I then removed, although I think this interface is still better

Reply:
As the handlebars thing is only used here, maybe the `HandleBarsExpander` can be here too ?
It does not really matters though